packages/xchain-midgard/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "xchain-midgard",
+  "version": "0.1.0-alpha",
+  "description": "Midgard module that  auto-generates all the client code using openapi-generator-cli",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf lib/**",
+    "build": "yarn clean",
```

Comment:
`build` is used to compile `TS` sources into `JS` and to provide a `lib` folder - all needed to publish package at npm - check other xchain-*  packages

Reply:
Thanks! will modify the script and add a tsconfig-build.json